Composer/plugins/azurePublish/src/index.ts

```diff
@@ -180,7 +180,7 @@ class AzurePublisher {
     const { name, environment, hostname, luisResource, language } = customizeConfiguration;
     try {
       // Perform the deploy
-      await this.azDeployer.deploy(name, environment, null, null, null, language, hostname, luisResource);
+      await this.azDeployer.deploy(name, environment, null, null, null, null, language, hostname, luisResource);
```

Comment:
The same problem with luBuilder, qnaBuilder. It's a disaster to have a long param list. Can we optimize the interface defined in this pkg `@bfc/botframeworkdeploy`?